modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-google-cloud-file-create-sc_{context}"]
+= Creating a storage class for Google Cloud File Storage
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+* Create a storage class specifying the network that you want to use.
++
+[source,yaml]
+.Example YAML file
+--
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: my-sc
+  uid: edabf3f4-7bdc-4d95-b682-4307d83327b2
+  resourceVersion: '55980'
+  creationTimestamp: '2022-08-10T17:26:28Z'
+  managedFields:
+    - manager: Mozilla
+      operation: Update
+      apiVersion: storage.k8s.io/v1
+      time: '2022-08-10T17:26:28Z'
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:allowVolumeExpansion': {}
+        'f:parameters':
+          .: {}
+          'f:network': {}
+        'f:provisioner': {}
+        'f:reclaimPolicy': {}
+        'f:volumeBindingMode': {}
+provisioner: pd.csi.storage.gke.io
```

Comment:
I think provisioner is not right here.

Reply:
Changed to filestore.csi.storage.gke.io. Thanks!